hack/run-e2e-test-suite.sh

```diff
@@ -58,7 +58,7 @@ set -e
 
 # The output XML needs some work because it adds "artifacts" as a test case.
 # Also, the suites doesn't have a name so, we need to add one.
-go install github.com/iblancasa/junitcli/cmd/junitcli@v1.0.2
+go install github.com/RH-QE-Distributed-Tracing/junitcli/cmd/junitcli@v1.0.3
```

Comment:
Who owns that org?

Reply:
The Distributed Tracing Quality Engineering Team from Red Hat.